google/cloud/compute/disks/v1/mocks/mock_disks_connection.h

```diff
@@ -54,9 +54,10 @@ class MockDisksConnection : public compute_disks_v1::DisksConnection {
       (override));
 
   MOCK_METHOD(
-      StatusOr<google::cloud::cpp::compute::v1::DiskAggregatedList>,
+      StreamRange<std::pair<std::string,
```

Comment:
You probably noticed, but the `,` for `std::pair< ... , ... >` is confusing the pre-processor.  You may be need a type alias or maybe use an extra parenthesis? 

PS: This was only detected by the `publish-compute-docs` build, which means we are not using these mocks (or any other generated mocks) in an example or test compiled by normal builds.